device-types/Digi/Digi-CM-48.yaml

```diff
@@ -5,6 +5,8 @@ slug: digi-cm-48
 part_number: '70001949'
 u_height: 1
 is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (6.40 lbs or 2.90 kg)